### PR TITLE
Initial Resmo Integration 

### PR DESCRIPTION
**Pull request by shuetisha**
### What does this PR do?

Adds an Resmo integration to Datadog

### Motivation

The Resmo is a continuous asset visibility, security, and compliance solution for cloud and SaaS. Integrate with your tools to start querying and securing your resources. Monitor them in one place, run security and compliance audits, and get alerted on violations.
Using our APIs, engineers are free to focus on building core functionality, rather than having to provision and maintain servers and cloud infrastructure.

What does Resmo offer to Datadog users?
- Collect and monitor all your Datadog assets on a single platform.
- Query your Datadog monitors, roles, permissions, API Keys, and more.
- Set up rules to continuously assess your Datadog resources.
- Set up and receive rule notifications based on your Datadog resource configurations.

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Comment by urseberry** [comment]
Docs review on https://datadoghq.atlassian.net/browse/DOCS-5067

File: resmo/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+## Overview
+[Resmo][1] is a continuous asset visibility, security, and compliance solution for cloud and SaaS. Integrate with your tools to start querying and securing your resources. Monitor them in one place, run security and compliance audits, and get alerted on violations.
```
Comment (buraizu) [inline resmo/README.md:2]:
```suggestion
[Resmo][1] is a continuous asset visibility, security, and compliance solution for cloud and SaaS. Integrate with your cloud and SaaS tools to start querying and securing your resources. Monitor them in one place, run security and compliance audits, and get alerted on violations.
```

File: resmo/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+Using our APIs, engineers are free to focus on building core functionality, rather than having to provision and maintain servers and cloud infrastructure.
```
Comment (buraizu) [inline resmo/README.md:3]: As I understand it, this integration is involved with monitoring and security -- could you clarify how it helps with provisioning and maintaining servers and cloud infrastructure?

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks for opening a new PR, I've just made one additional suggestion and posted a question.

File: resmo/README.md (hunk 2 above)
Comment (shuetisha) [reply resmo/README.md:3]: Hi @buraizu,

We defined it at the "Monitor them in one place, run security and compliance audits, and get alerted on violations."  section. If you have any suggestions I would gladly add them.

File: resmo/README.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+4. Return to your Resmo account and enter the copied keys into the Organization API Key and User App Key fields. <br />
+
+![Resmo Screenshot][7]
+
+5. Select a site from the dropdown menu on the last field.
+
+![Resmo Screenshot][8]
+
+6. Click the **create** button, and you're ready to query your Datadog resources.
```
Comment (buraizu) [inline resmo/README.md:32]:
```suggestion
6. Click the **Create** button, and you're ready to query your Datadog resources.
```

**Review by buraizu — CHANGES_REQUESTED** [review]

File: resmo/README.md (hunk 2 above)
Comment (shuetisha) [reply resmo/README.md]: Hi @buraizu,

I appreciate your help, changes are merged now.

**Review by emarsha94 — CHANGES_REQUESTED** [review]
Thanks for submitting your PR! Please let me know if you have any questions on the comments. Additionally, please include a logo in .svg format.

**Review by buraizu — DISMISSED** [review]
Thanks for implementing those changes, this looks good from a documentation perspective.

File: resmo/manifest.json [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  "manifest_version": "2.0.0",
+  "app_uuid": "0439ab8c-a8f5-4b68-88c1-7083eed72e0d",
+  "app_id": "resmo",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Integrate with Resmo to visualize and secure your Datadog resources.",
```
Comment (emarsha94) [inline resmo/manifest.json:11]:
```suggestion
    "description": "Integrate with Resmo to visualize and secure your Datadog resources",
```

File: resmo/CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+# CHANGELOG - Resmo
+
+## 1.0.0 / 2023-07-03
```
Comment (emarsha94) [inline resmo/CHANGELOG.md:3]:
```suggestion
## 1.0.0 / 2023-04-15
```

**Review by emarsha94 — CHANGES_REQUESTED** [review]

File: resmo/README.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
+## Overview
+[Resmo][1] is a continuous asset visibility, security, and compliance solution for cloud and SaaS. Integrate with your cloud and SaaS tools to start querying and securing your resources. Monitor them in one place, run security and compliance audits, and get alerted on violations.
+
+What does Resmo offer to Datadog users?
+- Collect and monitor all your Datadog assets on a single platform.
+- Query your Datadog monitors, roles, permissions, API Keys, and more.
+- Set up rules to continuously assess your Datadog resources.
+- Set up and receive rule notifications based on your Datadog resource configurations.
```
[the remark this answers is not in the document]
Comment (anawishnoff) [reply resmo/README.md:8]: To follow up on Emily's comment, I think what would be helpful here is if you described the specific data that users will now be able to access (in Datadog or Resmo), as well as any assets that might be available (i.e a Datadog dashboard, etc)

File: resmo/README.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+4. Return to your Resmo account and enter the copied keys into the Organization API Key and User App Key fields. <br />
+
+![Resmo Screenshot][7]
+
+5. Select a site from the dropdown menu on the last field.
+
+![Resmo Screenshot][8]
+
+6. Click the **Create** button, and you're ready to query your Datadog resources.
```
Comment (anawishnoff) [inline resmo/README.md:31]: This section will need to be updated to instruct the user to instead navigate to the Resmo tile in Datadog, and click Connect Accounts to kick off OAuth.

File: resmo/README.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+
+See [Resmo Datadog Resources][9] for the list of resources collected from your Datadog account.
+
+## Uninstallation
+
+- **In Resmo**: Go to **Settings > Integrations** and select the Datadog integration you wish to uninstall. <br />
+
+- You have two options:
+  1. To temporarily pause the integration, click the **Disable** button.
+  2. To permanently remove the integration, click the **Delete** button.
```
Comment (anawishnoff) [inline resmo/README.md:42]: These instructions will also need to be updated to take OAuth into account. See [our docs](https://docs.datadoghq.com/developers/integrations/oauth_for_integrations/#publish-the-oauth-client) for guidelines on OAuth uninstallation instructions.

File: resmo/manifest.json [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+    "title": "Resmo",
+    "media": [
+      {
+        "media_type": "image",
```
Comment (anawishnoff) [inline resmo/manifest.json:15]: Note that any images that do not have a 16:9 aspect ratio will be cut off. We require all images to fit into this aspect ratio. Please see our [media guidelines here](https://docs.datadoghq.com/developers/integrations/create_a_tile/?tab=buildatileontheintegrationspage#media-carousel).

File: resmo/manifest.json [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+        "caption": "Resmo Datadog integration page with site setup example.",
+        "image_url": "images/resmo-site-setup.png"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
```
[the remark this answers is not in the document]
Comment (anawishnoff) [reply resmo/manifest.json:53]: Following up on Emily's comment, please also add the submitted/queried data types to this field

Comment (anawishnoff) [inline resmo/manifest.json:53]:
```suggestion
      "Supported OS::macOS",
      "Offering::Integration"
```

**Comment by anawishnoff** [comment]
Hi @shuetisha, are you still working on this pull request? Let us know if there's anything we can help with or clarify. Thanks!

**Comment by anawishnoff** [comment]
Closing for now as per email conversation

**Pull request closed without merging**